# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: toolkit/components/telemetry/Histograms.json
# Commit: aecd7795068a
# Full Hash: aecd7795068a42ee91f3ea624e52233a32341199
# Author: Tim Huang <tihuang@mozilla.com>
# Date: 2022-07-20 09:59:35
# Regressor Bug: 1773701
# File Overlap Count: 1
# Description:
#   Bug 1773701 - Part 3: Add email tracking telemetry probes. r=dimi
#   
#   This patch adds two telemetry probes.
#   1. The counter of classified third-party email trackers in different
#      categories.
# ==============================================================================

diff -r d393067941bd -r aecd7795068a toolkit/components/telemetry/Histograms.json
--- a/toolkit/components/telemetry/Histograms.json	Tue Jul 19 20:04:24 2022 +0000
+++ b/toolkit/components/telemetry/Histograms.json	Tue Jul 19 20:04:25 2022 +0000
@@ -13369,6 +13369,22 @@
     "alert_emails": ["tihuang@mozilla.com", "pbz@mozilla.com", "seceng-telemetry@mozilla.com"],
     "bug_numbers": [1762374]
   },
+  "EMAIL_TRACKER_COUNT": {
+    "record_in_processes": ["main"],
+    "products": ["firefox", "fennec"],
+    "expires_in_version": "never",
+    "releaseChannelCollection": "opt-out",
+    "kind": "categorical",
+    "labels": [
+      "base_normal",
+      "content_normal",
+      "base_email_webapp",
+      "content_email_webapp"
+    ],
+    "description": "A count of the classified email trackers. ('base_normal' = The classified email tracker is in base category and loaded under a normal site, 'content_normal' = The classified email tracker is in content category and loaded under a normal site, 'base_email_webapp' = The classified email tracker is in base category and loaded under a known email webapp, 'content_email_webapp' = The classified email tracker is in content category and loaded under a known email webapp.)",
+    "alert_emails": ["tihuang@mozilla.com", "seceng-telemetry@mozilla.com"],
+    "bug_numbers": [1773701]
+  },
   "SERVICE_WORKER_LAUNCH_TIME_2": {
     "record_in_processes": ["main", "content"],
     "products": ["firefox", "fennec"],
